### ISSUE TITLE
Feature suggestion: Hide taskbar completely when using Auto-Hide

### ISSUE THREAD

**Issue by wontairr** [dremin/RetroBar#769]
With the Vista theme on and Auto-Hide enabled, it leaves a tiny bit of the top peeking out. personally it bothers me, and it would be nice to be able to have it completely disappear when not in use. (I attached an image)

![image](https://github.com/dremin/RetroBar/assets/28687890/6f99f5bc-523f-460b-ba75-a7c021687a2b)

**Comment by DanMiller299** [comment]
Yes, same here. I especially notice this on the XP Blue theme... is the additional blue bar supposed to be there? I don't think it is

![image](https://github.com/dremin/RetroBar/assets/163022909/e1634463-dd0e-4fd9-b7cf-f56b889d9d6e)

**Comment by t6productions** [comment]
I'm still having this problem even though it's marked as completed.

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/a8e4bec0-083f-4913-bef5-0400d1c212fb" />

**Comment by dremin** [comment]
> I'm still having this problem even though it's marked as completed.

You need to enable "Hide taskbar line when hidden with auto-hide" in RetroBar Properties > Advanced.
